pages/governance.md

```diff
@@ -14,12 +14,7 @@ for mathematical optimization based on the JuMP modeling language.
 
 ## Code of Conduct
 
-The JuMP community strongly values inclusivity and diversity. Everyone should
-treat others with the utmost respect. Everyone in the community must adhere to
-the [NumFOCUS Code of Conduct](https://numfocus.org/code-of-conduct) and the
-[Julia Community Standards](https://julialang.org/community/standards/), which
-reflect the values of our community. Both these links have information on how to
-confidentially report a violation.
+All interactions related to JuMP are moderated by our [Code of Conduct](/code-of-conduct).
```

Comment:
```suggestion
See [Code of Conduct](/code-of-conduct).
```
The statement about "all interactions related to JuMP" is ambiguous and not necessary.